docs/docs/recipes/integrate-logto/go-web.mdx

```diff
@@ -0,0 +1,347 @@
+---
+sidebar_label: Go Web
+---
+
+import postSignOutRedirectUriFigure from './assets/go-web-post-sign-out-redirect-uri.png';
+import redirectUriFigure from './assets/go-web-redirect-uri.png';
+import AppNote from './fragments/_app-note.mdx';
+import ConfigurePostSignOutRedirectUri from './fragments/_configure-post-sign-out-redirect-uri.mdx';
+import ConfigureRedirectUri from './fragments/_configure-redirect-uri.mdx';
+import FurtherReadings from './fragments/_further-readings.md';
+import GetAppSecret from './fragments/_get-app-secret.mdx';
+import SignInFlowSummary from './fragments/_web-sign-in-flow-summary.mdx';
+
+# Go Web: Integrate Logto
+
+<AppNote type="Traditional web" />
+
+## Add Logto SDK as a dependency
+
+:::tip
+The following demonstration is built upon the <a href="https://gin-gonic.com">Gin Web Framework</a>. You may also integrate Logto into other frameworks by taking the same steps.
+In the following code snippets, we assume your app is running on <code>http://localhost:8080</code>.
+:::
+
+Execute in the project directory:
+
+```bash
+go get github.com/logto-io/go
+```
+
+Add the `github.com/logto-io/go/client` package to your application code:
+
+```go
+// main.go
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	// Add dependency
+	"github.com/logto-io/go/client"
+)
+
+func main() {
+	router := gin.Default()
+	router.GET("/", func(c *gin.Context) {
+		c.String(200, "Hello Logto!")
+	})
+	router.Run(":8080")
+}
+```
+
+## Use sessions to store user authentication information
+
+In traditional web applications, the user authentication information will be stored in the user session.
+
+Logto SDK provides a `Storage` interface, you can implement a `Storage` adapter based on your web framework so that the Logto SDK can store user authentication information in the session.
+
+:::note
+We recommend not using cookie-based sessions, as user authentication information stored by Logto may exceed the cookie size limit.
+In this example, we use memory-based sessions. You can use Redis, MongoDB, and other technologies in production projects to store sessions as needed.
```

Comment:
```suggestion
In this example, we use memory-based sessions. You can use Redis, MongoDB, and other technologies in production to store sessions as needed.
```
nit: I don't think there is a production "project", only production "environment". So instead of `in production projects`, I prefer `in production environment` or just simply `in production`